panel/pane/image.py

```diff
@@ -82,7 +82,7 @@ def _get_properties(self):
         elif self.height is not None:
             width = int((self.height/height)*width)
             height = self.height
-        if self._is_url(self.object) and not self.embed:
+        if self.embed:
```

Comment:
Isn't this logic the wrong way around?

```suggestion
        if not self.embed:
```

Reply:
Right, I mistakened self.object as the entire image, but it's actually just the path.